pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -220,7 +223,9 @@ func TestCRIListPodStats(t *testing.T) {
 	checkCRIRootfsStats(assert, c2, containerStats2, &imageFsInfo)
 	checkCRILogsStats(assert, c2, &rootFsInfo, containerLogStats2)
 	checkCRINetworkStats(assert, p1.Network, infos[sandbox1.PodSandboxStatus.Id].Stats[0].Network)
-	checkCRIPodCPUAndMemoryStats(assert, p1, infos[sandbox1Cgroup].Stats[0])
+	if runtime.GOOS == "linux" {
```

Comment:
nit: move the check into the `checkCRIPodCPUAndMemoryStats` function, (i.e. return if GOOS != linux)

Reply:
Fixed